<launch>
  <node
    name="usb_cam"
    pkg="usb_cam"
    type="usb_cam_node"
    output="screen"
    required="true" >

    <param name="camera_name" value="logitech_cam_1"/>
    <param name="pixel_format" value="yuyv"/>

    
    
   </node>

  <node
    name="detector"
    pkg="detector_demo"
    type="blob_detector.py"
    output="screen"
    required="true" />
  
  <node
    name="display"
    pkg="detector_demo"
    type="display.py"
    output="screen"
    required="true" />

  <node
    name="viewer"
    pkg="rqt_image_view"
    type="rqt_image_view"
    required="true" />
</launch>
